src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -629,21 +633,24 @@ public void testPrintTreeOnMoreThanOneFile() throws Exception {
             assertEquals("", systemErr.getLog());
         });
 
-        Main.main("-t", getPath("checks/metrics"));
+        Main.main("-t", getNonPackagePath(
+                "checks/metrics"));
```

Comment:
`getNonPackagePath("checks/metrics"));` => `getPath("")`
We should use input directory, `main`, just like we require all input files in that directory.